### DISCUSSION TITLE
How to make a variation from a depth map.

### DISCUSSION THREAD

**Discussion by tykim9999** [thygate/stable-diffusion-webui-depthmap-script#60] [Q&A] [answered]
Inspired by this tweet. https://twitter.com/justLV/status/1605280983468584962

He made a great variations from a depth map. Do you guys know how to do it?

**Comment by thygate** [comment, marked as answer]
He's using the stable diffusion 2.0 [depth conditional model](https://github.com/Stability-AI/stablediffusion#depth-conditional-stable-diffusion)

For automatic1111's webui, see the [wiki](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features#depth-guided-model) section on depth-guided-model for instructions :

* download the [512-depth-ema.ckpt](https://huggingface.co/stabilityai/stable-diffusion-2-depth) checkpoint
* place it in models/Stable-diffusion
* grab the [config](https://raw.githubusercontent.com/Stability-AI/stablediffusion/main/configs/stable-diffusion/v2-midas-inference.yaml) and place it in the same folder as the checkpoint
* rename the config to 512-depth-ema.yaml
* select the new checkpoint from the UI

The depth-guided model will only work in img2img tab.

**Comment by thygate** [comment]
There is a blender addon that uses it to texture an entire scene at once.

**Reply by tykim9999** [reply]
Thank you! In img2img tab, the depth map as a input right? How's the denoising strength? I tried 0.7 but it fills it with all white.

**Reply by thygate** [reply]
I think I misunderstood, and you're supposed to just feed it a regular image, I get the bad output too when using an actual depthmap..

**Reply by tykim9999** [reply]
Thanks :) It works well with a regular image.
